src/pages/User/components/UserData/UserData.stories.js

```diff
@@ -0,0 +1,16 @@
+export default {
+  title: 'Pages/User/Components/UserData',
+  component: UserData,
+  argsTypes: {},
+};
+
+const UserDataTemplate = (args) => ({
+  components: { UserUserData },
+  setup() {
+    return { args };
+  },
+  template: '<user-data v-bind="args" />',
+});
+
+export const Default = UserDataTemplate.bind({});
+Default.args = {};
```

Comment:
I'm not sure but I think that you need to import the component in the history, is it not?